src/components/AnchorForCommentsOnly/BaseAnchorForCommentsOnly.tsx

```diff
@@ -62,7 +62,7 @@ function BaseAnchorForCommentsOnly({onPressIn, onPressOut, href = '', rel = '',
             role={CONST.ROLE.LINK}
             accessibilityLabel={href}
         >
-            <Tooltip text={href}>
+            <Tooltip text={!isLinkHasImage ? href : undefined}>
```

Comment:
Since we repeat this ternary down below on line 74, can we extract this out into a variable and use it both here and on line 74? That way the code is more DRY

Reply:
it's done